src/core/types/chains.ts

```diff
@@ -85,13 +165,16 @@ export const ChainNameDisplay = {
   [ChainId.zora]: 'Zora',
   [ChainId.mainnet]: 'Ethereum',
   [ChainId.hardhat]: 'Hardhat',
-  [ChainId.hardhatOptimism]: 'Hardhat Optimism',
+  [ChainId.hardhatOptimism]: chainHardhatOptimism.name,
   [ChainId.goerli]: chain.goerli.name,
   [ChainId.sepolia]: chain.sepolia.name,
-  [ChainId['optimism-goerli']]: chain.optimismGoerli.name,
-  [ChainId['bsc-testnet']]: 'BSC Testnet',
-  [ChainId['polygon-mumbai']]: chain.polygonMumbai.name,
-  [ChainId['arbitrum-goerli']]: chain.arbitrumGoerli.name,
-  [ChainId['base-goerli']]: chain.baseGoerli.name,
-  [ChainId['zora-testnet']]: chain.zoraTestnet.name,
+  [ChainId.holesky]: chainHoleski.name,
+  [ChainId.optimismGoerli]: chain.optimismGoerli.name,
+  [ChainId.optimismSepolia]: chainOptimismSepolia.name,
+  [ChainId.bscTestnet]: 'BSC Testnet',
+  [ChainId.polygonMumbai]: chain.polygonMumbai.name,
+  [ChainId.arbitrumGoerli]: chain.arbitrumGoerli.name,
+  [ChainId.arbitrumSepolia]: chainArbitrumSepolia.name,
+  [ChainId.baseGoerli]: chain.baseGoerli.name,
+  [ChainId.zoraTestnet]: chain.zoraTestnet.name,
```

Comment:
Noticed `Zora Goerli` had testnet appended here; not exactly sure where that originates. 

<img width="222" alt="Screenshot 2023-11-22 at 1 02 41 AM" src="https://github.com/rainbow-me/browser-extension/assets/4412473/f8a71387-20f6-4db1-8ac9-1b31a6cd5ca6">

Reply:
<img width="340" alt="image" src="https://github.com/rainbow-me/browser-extension/assets/12115171/6b4ec5b9-17fd-47e5-9c42-81c87a8f43ca">

Reply:
Nice, thank you!